docs/animations/Expressions.md

```diff
@@ -76,7 +76,7 @@ Welcome to the ExpressionBuilder classes! The ExpressionBuilder classes are a C#
 
 ## <a name="setting-up-the-expressionbuilder-classes-with-your-app"></a>Setting up the ExpressionBuilder classes with your app
 
-To use the ExpressionBuilder in your app, add the Microsoft.Toolkit.Uwp.UI.Animations nuget package to your project. Next, within your app project, make sure to add the using statement to leverage the ExpressionBuilder classes:
+To use the ExpressionBuilder in your app, add the Microsoft.Toolkit.Uwp.UI.Animations nuget package to your project. Next, within your app project, make sure to add the using directive to leverage the ExpressionBuilder classes:
 
 ```csharp
 using  Microsoft.Toolkit.Uwp.UI.Animations.Expressions;

```

Comment:
Noticed a quick fix we can make at the same time while I was looking at the preview, extra space we had left here:

```suggestion
using Microsoft.Toolkit.Uwp.UI.Animations.Expressions;
```